packages/cli/src/tests/utils/build.js

```diff
@@ -15,23 +14,31 @@ describe('build (runs slowly)', () => {
   before(async () => {
     // basically does what `zapier init` does
     tmpDir = getNewTempDirPath();
+    console.log(tmpDir);
     await copyDir(
       path.resolve(__dirname, '../../../../../example-apps/typescript'),
       tmpDir
     );
 
     // When releasing, the core version the example apps points can be still
-    // non-existent. Let's make sure  it points to the latest one available on
-    // npm.
-    const packageJsonPath = path.join(tmpDir, 'package.json');
-    const packageJson = JSON.parse(
-      fs.readFileSync(packageJsonPath, { encoding: 'utf8' })
+    // non-existent. Let's make sure it points to the local one.
+    const coreDir = path.resolve(__dirname, '../../../../core');
+    const corePackageJsonPath = path.join(coreDir, 'package.json');
+    const corePackageJson = JSON.parse(
+      fs.readFileSync(corePackageJsonPath, { encoding: 'utf8' })
     );
-    packageJson.dependencies[PLATFORM_PACKAGE] = await getPackageLatestVersion(
-      PLATFORM_PACKAGE
+    await runCommand('npm', ['pack'], { cwd: coreDir });
+    const coreVersion = corePackageJson.version;
+    const coreTarball = `${PLATFORM_PACKAGE}-${coreVersion}.tgz`;
+    const appPackageJsonPath = path.join(tmpDir, 'package.json');
+    const appPackageJson = JSON.parse(
+      fs.readFileSync(appPackageJsonPath, { encoding: 'utf8' })
     );
-    fs.writeFileSync(packageJsonPath, JSON.stringify(packageJson));
-
+    appPackageJson.dependencies[PLATFORM_PACKAGE] = path.resolve(
+      coreDir,
+      coreTarball
+    );
```

Comment:
I needed to change this test to not use the currently published version but a tarball of the local version or else changes to the typescript app that didn't match the published definitions would break it.